source/utils/timed-out.ts

```diff
@@ -85,11 +85,12 @@ export default (request: ClientRequest, delays: Delays, options: TimedOutOptions
 	}
 
 	if (typeof delays.socket !== 'undefined') {
+		const {socket} = delays;
```

Comment:
No need to extract `socket` for only one use. Same below.

Reply:
The reason I did this change is that both TypeScript (and me) is confused on whether issues due to time will occur, since the code below is defined within a callback. If the object will be always defined, maybe I can switch to using non-null assertions instead.

Reply:
Fixed.